Editor/Modules/IMeshModuleAnalyzer.cs

```diff
@@ -8,6 +8,7 @@ public interface IMeshModuleAnalyzer
     {
         void Initialize(ProjectAuditorModule module);
 
-        IEnumerable<ProjectIssue> Analyze(BuildTarget platform, AssetImporter assetImporter);
+        IEnumerable<ProjectIssue> Analyze(IProjectAuditorSettingsProvider projectAuditorSettingsProvider,
```

Comment:
Just a comment to point out this idea:

The modules where I knew so far that some analyzers require settings I provide the IProjectAuditorSettingsProvider interface. Using this and the platform the analyzers can retrieve the platform settings.

TODO:  We could do that for every analyzer interface to stay consistent, not just Mesh and Texture module/analyzers.